hasher-matcher-actioner/hmalib/lambdas/actions/action_performer.py

```diff
@@ -22,8 +28,20 @@ def perform_enque_for_review(match_message: MatchMessage):
 
 
 def perform_action(match_message: MatchMessage, action_label: ActionLabel):
-    # TODO implement
-    logger.debug("perform action")
+    action_performer = get_action_perfromers_config()[action_label]
+    action_performer.perform_action(match_message)
+
+
+def get_action_perfromers_config() -> t.Dict[ActionLabel, ActionPerformer]:
+    # TODO Should Read From s3 Configs table and determine which performer dynamically
```

Comment:
I built the dynamodb config thing for these types of usecases, probably overall easier for same general results.

Reply:
Yeah. I figured I would implement once your PR merges